WCON_format.md

```diff
@@ -379,7 +379,7 @@ In this example, we imagine a directory containing files labelled `filename_0.wc
     "files":{
          "this":"_2",
          "prev":["_1", "_0"],
-         "next":"_3"
+         "next":["_3"]
```

Comment:
I have commented on the brackets issue in a comment at the bottom.  In particular in this case, I've required arrays around previous and next, which I think in a way improves readability since it shows clearly that these entries accept multiple values, whereas "this" only has one value.

Certainly reasonable people can disagree on this point and if you feel it's less readable I can create an extra case in the schema to allow for single-valued "next" entries not to be wrapped in an array.